src/routes/transactions/__tests__/controllers/get-creation-transaction.transactions.controller.spec.ts

```diff
@@ -117,6 +116,9 @@ describe('Get creation transaction', () => {
             } as Response),
           );
         default:
+          console.log('************');
+          console.log(url, getChainUrl, chain, getCreationTransactionUrl);
+          console.log('************');
```

Comment:
👀

Reply:
Ouch 😣 Removed in [552a628](https://github.com/safe-global/safe-client-gateway/pull/2107/commits/552a6285b2bd06e7219b04a2888fb974a3f6df44)